sql/moz-fx-data-shared-prod/apple_ads_external/ios_app_campaign_stats_v1/query.sql

```diff
@@ -2,61 +2,63 @@ WITH daily_stats AS (
   SELECT
     date_day AS `date`,
     campaign_name AS campaign,
-    campaign_id AS campaign_id,
+    campaign_id,
     REGEXP_EXTRACT(
       campaign_name,
       r'Mozilla_(?:FF|Firefox)_ASA_(?:iOSGeoTest_)?([\w]{2,3})_.*'
     ) AS campaign_country_code,
     ad_group_name,
     ad_group_id,
-    DATE_TRUNC(date_day, WEEK) AS week_start,
-    FORMAT_DATE("%b", date_day) AS `month`,
+    -- DATE_TRUNC(date_day, WEEK) AS week_start, -- TODO: this doesn't appear to be used later on, should it be?
```

Comment:
@kwindau These fields do not appear to be referenced downstream, any objections against just removing this code?

Reply:
I would imagine Richard is probably using it in the Looker dashboard, I would leave it for now - thanks for asking!

Reply:
I use it some analysis and if removing it here does not lead to any material gain then I will suggest to please keep it, otherwise feel free to remove it

Reply:
Sorry, I might have been unclear in what I meant here. @soGaussian it appears these fields are not part of the final select statement hence the question whether I should just remove them from the CTE here.

Reply:
If so then remove it